x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/Expressions.java

```diff
@@ -21,6 +22,14 @@
 
 public final class Expressions {
 
+    public enum FunctionArgument {
+        one_arg,
```

Comment:
Is `one_arg` really needed? Can't we just assume if no ordinal is given, assume there's only one.
I guess that leads to overloaded methods (or passing null which is not ideal) so maybe rename it to 'IMPLICIT' or 'DEFAULT'?